index.html

```diff
@@ -1,51 +1,306 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Nimplay</title>
-    <link rel="shortcut icon" href="./Images/nimplayLogo.jpg">
-    <link rel="stylesheet" href="style.css">
-    
-</head>
-<body>
-  <header>           
+    <link rel="shortcut icon" href="./Images/nimplayLogo.jpg" />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!--Main Bar-->
+    <header>
       <div class="bar2">
-          <div class="logo">
-            <a href="#" id="name">Nimrod Acosta</a> 
-          </div>
-          <nav class="burger">
-              <button class="normalb">
-                <img src="./Images/Union.png" class="union" alt="">                    
-              </button>
-            </nav>
+        <div class="logo">
+          <a href="#" id="name">Nimrod Acosta</a>
+        </div>
+        <nav class="burger">
+          <button class="normalb">
+            <img src="./Images/Union.png" class="union" alt="" />
+          </button>
+        </nav>
       </div>
-      
-
-  </header>
-  <section class="continer">
+    </header>
+    <!--Section Home-->
+    <section class="continer">
       <h1 class="title">Hey There. I’m Nimrod</h1>
       <h2 class="subtitle">I’am a Software Developer</h2>
-      <p class="paragraph">Encoding for life. <br> 
-        Web Solutions and Mobile Applications</p>
+      <p class="paragraph">
+        Encoding for life. <br />
+        Web Solutions and Mobile Applications
+      </p>
       <ul class="social">
-          <li class="socialimg">
-            <img src="./Images/Github.png" alt="">
-          </li>
-          <li class="socialimg">
-            <img src="./Images/Icon-Linkedin.png" alt="">
-          </li>
-          <li class="socialimg">          
-            <img src="./Images/hand.png" alt="">
-          </li>
-          <li class="socialimg">
-            <img src="./Images/Icon-Twitter.png" alt="">
-          </li>
-          <li class="socialimg">
-            <img src="./Images/Icon-Medium.png" alt="">
-          </li>
-        </ul>                      
-  </section>    
-</body>
-</html>
\ No newline at end of file
+        <li class="socialimg">
+          <img src="./Images/Github.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/Icon-Linkedin.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/hand.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/Icon-Twitter.png" alt="" />
+        </li>
+        <li class="socialimg">
+          <img src="./Images/Icon-Medium.png" alt="" />
+        </li>
```

Comment:
- [Optional] Please use `<a>` to wrap each image in your li as there are suppose to be links to your social handles, Like so
```html
 <li class="socialimg">
   <a href ="#"> <img src="./Images/Icon-Medium.png" alt="" /> </a>
  </li>
```